.github/workflows/examples-infineon.yaml

```diff
@@ -37,7 +37,10 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: ghcr.io/project-chip/chip-build-infineon:54
+            image: ghcr.io/project-chip/chip-build-infineon:74
+            env:
+               # TODO: this should probably be part of the dockerfile itself
```

Comment:
should we have an Issue for this?

Reply:
I notified Infineon. Given that we currently have a bit over 1600 open SDK issues I am not sure if individual issues actually provide value, so I opted just for todo and slack communication for now.

I believe issues are only valuable today if they are part of a project or tracked in some way (e.g. tags). I have neither of them for this work.

Reply:
sure sounds good.